payment/src/main/java/org/killbill/billing/payment/core/sm/PaymentAutomatonDAOHelper.java

```diff
@@ -164,7 +165,7 @@ public String getPaymentProviderPluginName() throws PaymentApiException {
     }
 
     public PaymentPluginApi getPaymentPluginApi() throws PaymentApiException {
-        final String pluginName = getPaymentProviderPluginName();
+        final String pluginName = getPaymentProviderPluginName(true);
         return getPaymentPluginApi(pluginName);
```

Comment:
I would have expected the opposite: Since this is called when [running payment operations](https://github.com/killbill/killbill/blob/work-for-release-0.17.x/payment/src/main/java/org/killbill/billing/payment/core/sm/payments/PaymentOperation.java#L74), and that we *only* want to fetch deleted payment method for `GET` operations, then shouldn't this be `false`?